README.md

```diff
@@ -126,6 +132,14 @@ perform an incremental build. If you need to make a clean build you can just
 delete the RAMCloud directory and the contents of the RAMCloud-install directory
 and run the above commands again to rebuild it.
 
+# Debugging
```

Comment:
Where to place core files is personal preference and not something that I think belongs in the README.

Reply:
I'd rather have this covered in the README, and is likely useful to have when working with RAMCloud.

Reply:
We don't have to give an actual value for `kernel.core_pattern`; instead, we can just say you may need to change this, you should look at its value to see what it is, and point them at the [man page](http://man7.org/linux/man-pages/man5/core.5.html) which explains how different values work and how the '%' placeholders are expanded.

Reply:
done